.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [12, 14, 16]
+        node: [20, 22, 23]
```

Comment:
24 is out right, perhaps drop 23 in favour of 24?

Reply:
Good suggestion, done